content/chat/index.adoc

```diff
@@ -0,0 +1,79 @@
+---
+layout: simplepage
+title: Chat
+---
+
+The Jenkins community discusses various project-related topics in multiple channels on the http://www.freenode.net[Freenode] IRC network.
+
+For information on how to connect, see the http://freenode.net/[Freenode] website.
+
+== Channels
+
+=== `#jenkins`
+
+This is the *primary channel* for discussions about Jenkins.
+In you have questions about Jenkins, you'll probably want to ask them here.
+
+http://webchat.freenode.net/?channels=jenkins[Join using a web-based IRC client].
+
+NOTE: if you have a question about Jenkins, just ask it -- don't ask whether you can ask a question.
+It's common to not receive an answer for quite a while if no active contributors are currently checking IRC.
+In that case, just stick around, and periodically (every few hours) ask your question again.
+
+Conversations in this channel are automatically archived to both http://echelog.com/?jenkins[echelog] and https://botbot.me/freenode/jenkins/[botbot.me].
+
+We use the *jenkins-admin* bot to automate common administrative operations.
+Its use is limited to users with _voice_ or _op_ in this channel.
+link:/projects/infrastructure/ircbot/[Learn more].
+"Core" project contributors typically have _op_ (@) in this channel.
+Established contributors typically have _voice_ (+) in this channel.
+
+=== `#jenkins-community`
+
+The channel to help organize events, documentation and more "meta conversations" around the Jenkins project.
+We don't discuss Jenkins -- the software -- here.
+
+[[meeting]]
+=== `#jenkins-meeting`
+
+We conduct bi-weekly project meetings in this channel.
+These meetings are open to everyone.
+link:/project/governance/#meeting[[Learn more about the project governance meeting].
+
+Meetings are logged, and the channel is unused the rest of the time.
+We use the *robobutler* to run these meetings.
+See the link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Meeting+Agenda[about project meetings] for more details.
+
+=== `#jenkins-infra`
+
+Discussions of the Jenkins project infrastructure, i.e. most services running on +jenkins.io+, +jenkins-ci.org+, and related domains.
+
+=== `#jenkins-ux`
+
+Discussions on Jenkins UX efforts such as link:/projects/blueocean/[Blue Ocean].
```

Comment:
The channel still exists, and there's still people. So I see no particular need to remove it from the list.

Regarding Gitter in general: The presence of people interested in Jenkins doesn't make it an official communications channel of the Jenkins project.